MenuSystem.h

```diff
@@ -217,6 +217,46 @@ class MenuItem : public MenuComponent {
     virtual Menu* select();
 };
 
+class TextMenuItem : public MenuItem {
```

Comment:
What benefit does a `TextMenuItem` have over using multiple `MenuItem`'s and cycling between them?

Reply:
I'm also not a fan of the name since a `MenuItem` is really a menu item with some text. This is more like a combo box or select box. So if it is useful, I'd prefer a different name.